packages/did-auth-siop-op-authenticator/README.md

```diff
@@ -0,0 +1,27 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>DID Auth SIOP OP Authenticator (Typescript) 
+  <br>
+</h1>
+
+---
+
+**Warning: This package still is in every early development. Breaking changes without notice will happen at this point!**
+
+---
+
+A Veramo plugin to authenticate using the Self Issued OpenID Provider v2 (SIOPv2)
```

Comment:
Could we add a bit more documentation and link to our SIOP README, which is quite extensive?

Ideally this README refers to the SIOP README for most and then lists the differences and/or things to consider in the context of Veramo. Also have some example code and calls would be helpful

Reply:
added a better readme